starlite/handlers/utils.py

```diff
@@ -5,6 +5,9 @@
 from starlite.datastructures.cookie import Cookie
 from starlite.datastructures.response_header import ResponseHeader
 
+__all__ = ("narrow_response_cookies", "narrow_response_headers")
```

Comment:
these utils are used on different layers of the app - should they be here or in an "internal_utils.py" package? @starlite-api/maintainers